plugins/kubernetes/app/assets/javascripts/kubernetes/filters/default_filter.js

```diff
@@ -0,0 +1,7 @@
+samson.filter('default', function() {
+
+  return function(input, str) {
+    return _.isUndefinedOrEmpty(input) ? str : input;
```

Comment:
Same as above. If I return `''` instead of the original value when `undefined`, the above filter will have to deal with the condition `defined but empty` as well, because they've been chained in the HTML. I know that's what you suggested above, but I think although they're being used together, each filter should be agnostic of the other, meaning, they should be applied when possible but return the original value otherwise. This way, they can even be reused anywhere.

What do you think?